packages/data-point/lib/core/core.js

```diff
@@ -92,6 +93,7 @@ function create (spec) {
   // supports currying
   manager.resolve = Transform.resolve(manager)
   // does not support currying
+  manager.reducer = _.partial(Transform.transform, manager)
```

Comment:
we don't need this line

Reply:
One of the unit tests in core.test.js fails without this line of code

Reply:
Um, are you talking about the test that you added specifically for testing this line? All the tests on master are currently passing.